templates/kubeflow/consulting.html

```diff
@@ -25,8 +25,8 @@ <h4 class="u-sv3">
         >
         <a
           class="p-link--external"
-          href="{{ ASSET_SERVER_URL }}62e71815-Kubernetes-for-the-Enterprise.pdf"
-          >Read our consulting datasheet</a
+          href="https://ubuntu.com/engage/starting-with-ai?utm_source=bubble&utm_campaign=CY19_DC_AI_Whitepaper_AIConsulting"
```

Comment:
here you can use the relative path: "/engage/starting-with-ai?utm_source=bubble&utm_campaign=CY19_DC_AI_Whitepaper_AIConsulting"